### PR TITLE
PLF-6687 Remove the hardcode of portal site name in source code

### PR DESCRIPTION
**Pull request by trongtt**

### CODE REVIEW

File: application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java [hunk 1]
```diff
@@ -55,14 +57,18 @@ private String createOrEditPage(String creator, String parentTitle, String title
   {
     String wikiType = PortalConfig.PORTAL_TYPE;
     String wikiOwner = "intranet";
```
Comment (nttuyen) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java]: Done

File: application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java [hunk 2]
```diff
@@ -55,14 +57,18 @@ private String createOrEditPage(String creator, String parentTitle, String title
...
-    String path = "";
+    PortalRequestContext pContext = Util.getPortalRequestContext();
+    if (pContext != null) {
+      wikiOwner = pContext.getPortalOwner();
+    }
```
Comment (nttuyen) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java]: agree and I updated the PR

File: application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java [hunk 3]
```diff
@@ -53,16 +57,22 @@ protected String createSpacePage(String creator, String title, String content, S
...
-    {
+    if (spaceGroupId != null) {
       wikiType = PortalConfig.GROUP_TYPE;
       wikiOwner = "/" + spaceGroupId;
+    } else {
+      if (pContext != null) {
+        wikiOwner = pContext.getPortalOwner();
+      } else {
+        wikiOwner = CommonsUtils.getService(UserPortalConfigService.class).getDefaultPortal();
+      }
```
Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java]: Could be replaced by CommonsUtils.getCurrentSite().

Comment (nttuyen) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java]: updated

Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java]: I meant, we could replace the whole block. Isn't that what CommonsUtils.getCurrentSite() already does ?

Comment (nttuyen) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java]: ah.. yes, I updated like that

File: application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java [hunk 4]
```diff
@@ -1,6 +1,11 @@
 package org.exoplatform.chat.portlet.chat;
 
+import org.exoplatform.commons.utils.CommonsUtils;
+import org.exoplatform.portal.application.PortalRequestContext;
+import org.exoplatform.portal.config.UserPortalConfigService;
 import org.exoplatform.portal.config.model.PortalConfig;
+import org.exoplatform.portal.mop.SiteKey;
+import org.exoplatform.portal.webui.util.Util;
```
Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/chat/WikiService.java]: There are useless imports.

**Pull request merged**